test/LongVectorLowering/globals.ll

```diff
@@ -0,0 +1,26 @@
+; RUN: clspv-opt --LongVectorLowering --instcombine %s -o %t
+; RUN: FileCheck %s < %t
+
+target datalayout = "e-p:32:32-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024"
+target triple = "spir-unknown-unknown"
+
+@a_global = external addrspace(2) constant <8 x float>, align 32
```

Comment:
It would be good to have a version of this test where the global variable has an initializer.

Reply:
Good idea. I've just pushed a change to address that.